src/main/resources/ath-container/run.sh

```diff
@@ -39,50 +47,33 @@ fi
 
 browser=$1
 war=$2
+extra_args=
+
 if [[ ! -f $war ]]; then
 	case "$war" in
 	latest)
-		war=jenkins-latest.war
-		url=https://updates.jenkins.io/latest/jenkins.war
+		# Maven will download this in the process-test-resources phase
+		war=target/jenkins-war.war
 		;;
 	lts)
-		war=jenkins-lts.war
-		url=https://updates.jenkins.io/stable/latest/jenkins.war
+		# Maven will download this in the process-test-resources phase
+		war=target/jenkins-war.war
+		extra_args='-Plts'
+		;;
+	*.war)
+		download "$war" "jenkins.war" || exit 1
+		war=jenkins.war
+		;;
```

Comment:
Due to the `[[ ! -f $war ]]; then` a few lines above, this is the case statement for URLs, not local file paths.